.github/workflows/BuildAndDeploy.yaml

```diff
@@ -20,10 +23,23 @@ jobs:
       - name: Build
         run: npm run build
```

Comment:
do we have the docs link redirect for production already in place? If yes, we need to build it with production right?

Reply:
Currently it doesn't check if it's in production, it checks whether it's in development which seems to work as is, should this be changed?